README.rst

```diff
@@ -13,9 +13,11 @@ Compatible with Django's `storage API <https://docs.djangoproject.com/en/dev/ref
 Supported Versions
 ==================
 
-Python 2.6/2.7 with Django 1.4+
-Python 3.2/3.3/3.4 with Django 1.5+
-Python 3.5/3.6 with Django 1.7+
+- Python 2.7 with Django 1.11
+- Python 3.5/3.6/3.7 with Django 1.11+
+- Python 3.8 with Django 2.2+
+
+In general, we follow `Python's supported versions <https://devguide.python.org/#status-of-python-branches>`_ and `Django's supported versions <https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django>`_. Any major change in version support will be released as a new major version.
```

Comment:
> Python 2.7

https://pythonclock.org/

![image](https://user-images.githubusercontent.com/26336/71307770-ba9c9500-23b8-11ea-921f-f847777258aa.png)

We are nearing the end of an era

> Django 1.11 (anything <2.2)

They are advising 2.2 and on. It'd probably simplify the support (https://docs.djangoproject.com/en/3.0/releases/3.0/)

I am going to bring this conversation up at Jazzband also. There is a lot of old API code pull out and travis runs. 😆

Reply:
Django 1.11 technically has support until [April 2020](https://www.djangoproject.com/download/#supported-versions). At that point, I would feel comfortable dropping support for Python 2.7 and Django 1.11 entirely. 

Once we hit April 2020, we should (IMO, open to debate) support:

* Django 2.2 LTS
* Django 3.0
* Python 3.5 (for Django 2.2 LTS users)
  * It seems reasonable to assume someone on Django 2.2 LTS is using Python 3.5 because it's the most recent version of Python they can get. The maintenance burden for additional 3.5 to 3.8 versions is "zero" so I'm cool supporting them.
* Python 3.6, 3.7, 3.8
  * The maintenance burden for the additional 3.6 to 3.8 versions is "zero" so I'm cool supporting them